plugins/mac-capture/window-utils.m

```diff
@@ -60,8 +60,10 @@ bool find_window(cocoa_window_t cw, obs_data_t *settings, bool force)
 
 		NSNumber *window_id = (NSNumber *)dict[WINDOW_NUMBER];
 		cw->window_id = window_id.intValue;
+		cw->owner_pid = (NSNumber *)dict[OWNER_PID];
```

Comment:
See above, this `NSNumber` is likely going to be released at a "random" point in time, leaving you with a dangling pointer (also all the other places where `NSFoo` are retained or release aren't handled for this)

Reply:
Done